css/layout.css

```diff
@@ -226,3 +226,15 @@ footer nav a {
 #menu-dropdown:target {
     display: none;
 }
+
+#nav {
+	animation: show .8s cubic-bezier(.45,.05,.55,.95);
```

Comment:
Not so sure about this animation, I think there's already enough things moving on page load, the hero section moves and text also flickers probably because of the *FOUC*(flash of unstyled content) produced by the font loading. Maybe just make it more subtle like animating opacity? we can also solve FOUC adding a class to the body that makes text transparent and remove it once [fonts load](https://developer.mozilla.org/en-US/docs/Web/API/Document/fonts) to avoid the visual flickering.